examples/redshift/README.md

```
@@ -0,0 +1,84 @@
+type: redshift
+host: mycluster.abc123.us-east-1.redshift.amazonaws.com
+port: 5439
+database: dev
+username: YOUR_USER
+password: YOUR_PASSWORD
+```
+
+Or as a connection string, using `redshift-connector` (the default driver —
+supports IAM auth and Redshift Serverless, not just password auth):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)